src/components/Application-Body/JobsList/jobsList.tsx

```diff
@@ -1,22 +1,46 @@
 import * as React from 'react';
+import ListIcon from '@material-ui/icons/List';
+import BlockIcon from '@material-ui/icons/Block';
+
 import styles from './jobsListStyle';
 import Job from './job';
-
 import { Advertisement  } from '../../../types/Advertisements';
 
 
 interface JobsListProps {
-    ads: Advertisement []
-
+    ads: Advertisement[];
+    isFiltered: boolean;
 }
 
 const JobsList: React.FC<JobsListProps> = (props): JSX.Element => {
-    const { ads } = props;
+    const { ads, isFiltered } = props;
     const classes = styles();
 
+    const renderJobs = (): JSX.Element => {
+        if (ads.length === 0) {
+            if (isFiltered) {
```

Comment:
I think we never get here....

isFiltered = searchValue not empty && ads.length !==0.

But in line 20, in the if clause, you check if length is 0....

So even if DB was empty we wouldn't have displayed that error.

Contact me if something is not clear 😃

Reply:
You explained in phone.. I didn't notice that there are 2 variables. Matter is closed 😃